appd.adoc

```diff
@@ -171,7 +167,7 @@ z(n,k,j,i) = a(k)*ztop + b1(k)*zsurf1(n,j,i) + b2(k)*zsurf2(n,j,i)
               
 ----
 
-where **`z(n,k,j,i)`**is the height above the geoid (approximately mean sea level) at gridpoint **`(k,j,i)`**and time **`(n)`**, **`ztop`**is the height of the top of the model, and **`a(k)`**, **`b1(k)`**, and **`b2(k)`**are the dimensionless coordinates which define hybrid level **`k`**. **`zsurf1(n,j,i)`**and **`zsurf2(n,j,i)`**are respectively the large and small parts of the topography. See Shaer et al [<<sch02,SCH02>>]for details.
+where **`z(n,k,j,i)`**is the height above the geoid (approximately mean sea level) at gridpoint **`(k,j,i)`**and time **`(n)`**, **`ztop`**is the height of the top of the model, and **`a(k)`**, **`b1(k)`**, and **`b2(k)`**are the dimensionless coordinates which define hybrid level **`k`**. **`zsurf1(n,j,i)`**and **`zsurf2(n,j,i)`** are respectively the large and small parts of the topography. See Shaer et al [<<sch02,SCH02>>] for details.
```

Comment:
It'd be nice to have a space after `zsurf1` in the prose, just like you've done for `zsurf2`.